### PR TITLE
feat: add results and page metadata to filter bar

### PR DESCRIPTION
**Pull request by jaredcwhite**
This PR addresses #756 

- [x] Addresses the issue in full
- [ ] Addresses only certain aspects of the issue

## Description

This moves the region which had the "Filters" button down into the right listings column, and adds Total Results & Page X of Y metadata.

## How Can This Be Tested/Reviewed?

View the Listings page, tweak filters, etc.

**Note:** we might want to do a pass of testing accessibility for this…not sure if there's anything specific we need to do for better markup.

## Author Checklist:

- [ ] Added QA notes to the issue with applicable URLs
- [x] Reviewed in a desktop view
- [x] Reviewed in a mobile view
- [ ] Reviewed considering accessibility
- [ ] Added tests covering the changes
- [ ] Made corresponding changes to the documentation
- [ ] Ran `yarn generate:client` and/or created a migration when required

## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates

### CODE REVIEW

**Review by emilyjablonski — CHANGES_REQUESTED** [review]
Screenshot 2024-08-08 at 12 07 49 PM
Screenshot 2024-08-08 at 12 07 45 PM
Screenshot 2024-08-08 at 12 07 31 PM
Screenshot 2024-08-08 at 12 07 23 PM

File: sites/public/src/components/listings/ListingsCombined.module.scss [hunk 1]
```diff
@@ -149,3 +141,17 @@
     }
   }
 }
+
+.listings-list-wrapper {
+  --padding: 2.5rem;
```
Comment (emilyjablonski) [inline sites/public/src/components/listings/ListingsCombined.module.scss:146]: `--seeds-s10`

File: sites/public/src/components/listings/search/ListingsSearch.module.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
+.listings-vars {
+  // This is a not-ideal way to do "fill window minus header and filter modal" however
+  // I can't find another way to do this.
+  // TODO: update header + filter modal to a not-magic number
+  --listings-component-height: calc(100vh - 183px);
+  --listings-component-half-height: calc(var(--listings-component-height) / 2);
+  --listings-list-width: 600px;
+  --swipe-area-height: var(--bloom-s14);
```
Comment (emilyjablonski) [inline sites/public/src/components/listings/search/ListingsSearch.module.scss:8]: `--seeds-s14`

File: sites/public/src/components/listings/search/ListingsSearch.module.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  --background-color: var(--bloom-color-white);
```
Comment (emilyjablonski) [inline sites/public/src/components/listings/search/ListingsSearch.module.scss:9]: `--seeds-bg-color-surface`

**Comment by emilyjablonski** [comment]
Screenshot 2024-08-08 at 12 43 06 PM
We're also showing 1 of 0 in an empty state

**Comment by jaredcwhite** [comment]
Thanks @emilyjablonski for the feedback! I'll work on making changes.

**Comment by jaredcwhite** [comment]
@emilyjablonski I think it has been tightened up now. One change I didn't make was that I kept the text button color intact on mobile. I'm not sure why in the Figma the button shows the same dark gray text color as regular text. We can ask design, but that seems out of step with the design system to me.

I also noticed some horizontal padding issues with some other content in the sidebar, but that was present before this ticket. We can decide if we want to address that here or separately.

**Comment by jaredcwhite** [comment]
Thanks @emilyjablonski for the additional review! I think we're OK regarding the Total Results font size – I took screenshots of the page and the Figma design and compared the sizes and it's looking identical (at least on my machine in Safari).

**Pull request merged**